genome_browser/_tracks.py

```diff
@@ -155,7 +155,7 @@ def plot(self, ax=None):
 
         # The non-overlapping disjoint intervals are computed using the
         # logic which priortizes first position of interval, then length.
-        levels = list(disjoint_bins(self.intervals))
+        levels = list(disjoint_bins(self.interval_pretty_sort))
```

Comment:
Without this, `disjoint_bins` would get thrown off:

![widths_borked](https://user-images.githubusercontent.com/13955209/169541167-899707d9-064d-40f9-a0e3-a98fb6df48f7.png)